tests/test_main.py

```diff
@@ -2156,3 +2157,15 @@ class Model(BaseModel):
         'properties': {'x': {'title': 'X', 'anyOf': [{'type': 'integer'}, {'type': 'string'}]}},
         'required': ['x'],
     }
+
+
+def test_annotated_class():
+    class PydanticModel(BaseModel):
+        foo: str = '123'
+
+    PydanticAlias = Annotated[PydanticModel, 'bar baz']
+
+    pa = PydanticAlias()
+    pa.__doc__ = 'qwe'
```

Comment:
```suggestion
    pa = PydanticAlias()
    assert isinstance(pa, PydanticModel)
```

I think it makes the test a bit clear, it's not immediately obvious what the result of instantiating something wrapped with `Annotated` should be.

Reply:
done.